grossmont/computer-science-and-cybersecurity-organization/sections/footer/footer.html

```diff
@@ -0,0 +1,51 @@
+<style>
+.footer{
+    font-family: 'Sen', sans-serif;
+    text-align: center;
+    background-color: #363537;
+    color: white;
+    position: absolute;
+    bottom: 0;
+    width: 100%; 
+}
+a{
+    text-decoration: none;
+    text-decoration-color: none;
+    color: #ef2d56
+}
+a:hover{
+    color: #a61c3a
+}
+</style>
+
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Sen:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./header.css">
+</head>
+
+
+
+<body>
+<footer class="footer" >
+<nav>
+        <p>
+        <a href="/grossmont/computer-science-and-cybersecurity-organization/pages/about/about.html" target="_parent">
+            [About Us]
+        </a>
+        ,
+        <a href="https://github.com/khiemgluong/grossmont-cybersecurity-club" target="_blank">
+            [Source Code]
+        </a>
+        </p>
+    <p class="text">
+        Join us on
+        <a href="https://discord.gg/48mvhtCASz" target="_blank"> Discord </a>
+    </p>
```

Comment:
Hm, I definitely put those as more of "temporary links" since I think people would have more ideas of what we could utilize the footer for. However, I think we could remove the Discord linked in the contact page since it is a bit out of place in terms of styling. Maybe we could create a file for "about us" and shows each member who's contributed to the creation of the site and says what they've done (eg. Jaye helped with color palette, etc.). However, i think we could find something else to use instead of "About Us" for the footer. I'd be open to suggestions. 

Will fix the old link next time I get to working on this.